src/dlgConnectionProfiles.cpp

```diff
@@ -282,6 +279,16 @@ dlgConnectionProfiles::dlgConnectionProfiles(QWidget* parent)
         mCustomIconColors.append(QColor::fromHsv(i, 192, 255));
         mCustomIconColors.append(QColor::fromHsv(i, 128, 255));
     }
+
+    mSearchTextTimer.setInterval(1000);
```

Comment:
For easier reading!

```suggestion
    mSearchTextTimer.setInterval(1s);
```

At the top, add:

```
#include <chrono>
using namespace std::chrono_literals;
```

Reply:
I'm getting Deja Vu all over again about this (`std` time intervals) - I'm not a fan of them but I guess i can do it... :slightly_frowning_face: